packages/matrix-identity-server/src/db/index.ts

```
@@ -1,276 +1,225 @@
-  getAll: GetAll<T>
-  getHigherThan: Get<T>
-  match: Match<T>
-  update: Update<T>
-  updateAnd: UpdateAnd<T>
-  deleteEqual: DeleteEqual<T>
-  deleteEqualAnd: DeleteEqualAnd<T>
-  deleteLowerThan: DeleteLowerThan<T>
-  deleteWhere: DeleteWhere<T>
-  close: () => void
+  ready: Promise<void>;
+  createDatabases: (conf: Config, ...args: any) => Promise<void>;
```

Biome (2.4.9)

[error] Unexpected any. Specify a different type.

(lint/suspicious/noExplicitAny)